lib/auth0/api/v2/jobs.rb

```diff
@@ -20,15 +20,19 @@ def get_job(job_id)
         # @see https://auth0.com/docs/api/v2#!/Jobs/post_users_imports
         # @param users_file [file] A file containing the users to import.
         # @param connection_id [string] Database connection ID to import to.
+        # @param options [hash] upsert / external_id / send_completion_email.
         #
         # @return [json] Returns the job status and properties.
-        def import_users(users_file, connection_id)
+        def import_users(users_file, connection_id, **options)
           raise Auth0::InvalidParameter, 'Must specify a valid file' if users_file.to_s.empty?
           raise Auth0::InvalidParameter, 'Must specify a connection_id' if connection_id.to_s.empty?
 
           request_params = {
             users: users_file,
-            connection_id: connection_id
+            connection_id: connection_id,
+            upsert: options[:upsert].nil? ? false : options[:upsert],
```

Comment:
In this library, we typically use `hash.fetch` for values:

https://github.com/auth0/ruby-auth0/blob/master/lib/auth0/api/v2/users.rb#L30

Reply:
Thank you for the review.

It was simpler to use `hash.fetch`